src/components/pages/home/index.tsx

```diff
@@ -11,15 +12,35 @@ import Link from 'next/link'
 import {track} from 'utils/analytics'
 import Jumbotron from 'components/pages/home/jumbotron'
 import {Jumbotron as HolidayReleaseJumbotron} from 'components/pages/20-days-of-egghead/course-grid'
+import toast, {Toaster} from 'react-hot-toast'
 
 const Home: React.FC<any> = ({data, jumbotron, location}) => {
+  const router = useRouter()
+
+  React.useEffect(() => {
+    const {query} = router
+    if (query.message) {
+      toast(query.message as string, {
+        icon: '✅',
+        duration: 5000,
+      })
+    }
+  }, [router])
```

Comment:
`/learn` page checks if a message is set in the url and displays a toast pop up if so.

Currently set toast to 5 seconds, maybe needs to be lower?